packages/linter/src/builders/lint/lint.impl.ts

```diff
@@ -110,10 +111,13 @@ async function run(options: Schema, context: BuilderContext): Promise<any> {
   context.logger.info(formattedResults);
 
   if (options.outputFile) {
-    writeFileSync(
-      path.join(context.workspaceRoot, options.outputFile),
-      formattedResults
+    const pathToFile = path.join(context.workspaceRoot, options.outputFile);
+    const fileDirectoryPath = pathToFile.substring(
+      0,
+      pathToFile.lastIndexOf('/')
     );
+    createDirectory(fileDirectoryPath);
```

Comment:
Can't you just use `path.dirname`?

```suggestion
    createDirectory(path.dirname(pathToFile));
```

Does `createDirectory` handle the case where the folder already exists?

Reply:
ah yes, thanks for the suggestion 👍 much simpler! And yes, `createDirectory` is fine with a directory already being there